ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -20,6 +20,13 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "COMPLY_ON_BEHALF",
```

Comment:
yes, this makes sense now I have the context of what has happened in CMO. I wanted CMO to be of Order type, but it has it's own type in Java. So there will be code differences in the callbacks

Reply:
Yes these will need to be different. Unless we can maybe refactor some of the methods to return List<Element<Direction>> rather than adding to SDO. Something to look into when I am doing CMO compliance

Reply:
I have changed COMPLY_ON_BEHALF -> COMPLY_ON_BEHALF_SDO. There are other events that will also need to change. I shall address these elsewhere